pygpudrive/datatypes/observation.py

```diff
@@ -25,10 +26,11 @@ def __init__(self, self_obs_tensor: torch.Tensor):
         self.speed = self_obs_tensor[:, :, 0]
         self.vehicle_length = self_obs_tensor[:, :, 1]
         self.vehicle_width = self_obs_tensor[:, :, 2]
-        self.rel_goal_x = self_obs_tensor[:, :, 3]
-        self.rel_goal_y = self_obs_tensor[:, :, 4]
-        self.is_collided = self_obs_tensor[:, :, 5]
-        self.id = self_obs_tensor[:, :, 6]
+        self.vehicle_height = self_obs_tensor[:, :, 3]
```

Comment:
Is this the object height or its z position?

Reply:
Oh seems like it's actual height

Reply:
Yep!